BTSPocket/Controllers/Login/LoginViewController.swift

```diff
@@ -36,6 +36,7 @@ class LoginViewController: UIViewController {
         self.textPassword.delegate = self
         self.textEmail.returnKeyType = .next
         self.textPassword.returnKeyType = .done
```

Comment:
Less code is better.  Config return key type  via storyboard and remove this line